docs/zh/guide/ssh.md

```diff
@@ -0,0 +1,60 @@
+# ssh
+
+EdgeMesh 的socks5代理提供了节点之间通过socks5代理进行ssh登录访问的能力，本章节会对此功能进行详细介绍。
+
+## SSH代理工作原理
+
+![edgemesh-socks5-proxy](/images/guide/em-sock5.png)
+
+1. 客户端通过代理发起远程登录请求，流量将会被转发到代理服务中
+2. 在代理服务器中解析目的主机名和端口将其转换成远程服务器IP地址
+3. 通过原有Tunnel模块的P2P打洞功能将流量转发至目标机器
+4. 通道建立后对远端流量的响应返回给SSH客户端，完成通道建立
+
+## 配置
+
+```shell
+$ build/agent/kubernetes/edgemesh-agent/05-configmap.yaml
+  modules:
+    ..
+    edgeProxy:
+      ..
+      socks5Proxy:
+        enable: true
+        listenPort: 10800
+    ..
+```
+
+::: warning
+更改后需要重新启动edgemesh-agent
+:::
+
+
+## 使用
+
+**由于节点的IP可以重复，所以只支持通过节点名称进行连接**
+
+```shell
+$ kubectl get node
+NAME                                  STATUS   ROLES                         AGE    VERSION
+master                                Ready    control-plane,master,worker   22d    v1.21.5
+edge-node-1002                        Ready    agent,edge                    2d4h   v1.19.3-kubeedge-v1.7.2
+
+
+$ssh -o "ProxyCommand nc --proxy-type socks5 --proxy 169.254.96.16:10800 %h %p" root@edge-node-1002
+
+ECDSA key fingerprint is SHA256:uPUzjIPK+zvu8ymzrkd0IWSsmrs2r/Hl72iendYniVY.
+ECDSA key fingerprint is MD5:ca:80:85:84:bd:09:a6:fd:d9:ba:73:1c:b5:7c:f6:ae.
+Are you sure you want to continue connecting (yes/no)? yes
+root@1.1.1.1's password: 
+
+Activate the web console with: systemctl enable --now cockpit.socket
+
+Last failed login: Wed Dec 15 14:45:06 CST 2021 from 192.168.1.128 on ssh:notty
+There was 1 failed login attempt since the last successful login.
+Last login: Wed Dec 15 14:07:04 2021 from 192.168.1.128
+[root@edge-node-1002 ~]#
+```
+::: warning
```

Comment:
warning -> info

Reply:
Thank you for your review. I have modified them